package.json

```diff
@@ -37,21 +37,21 @@
   "homepage": "https://github.com/coatue-oss/react2angular#readme",
   "peerDependencies": {
     "@types/angular": ">=1.5",
-    "@types/prop-types": ">=15",
-    "@types/react": ">=16",
-    "@types/react-dom": ">=16",
+    "@types/prop-types": ">=18",
```

Comment:
When trying to test install this I get
```
npm WARN Could not resolve dependency:
npm WARN peer @types/prop-types@">=18" from react2angular@7.0.1
npm WARN node_modules/react2angular
npm WARN   react2angular@"domotz/react2angular#react18" from the root project
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @types/prop-types@>=18.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

So basically `@types/prop-types": ">=18"` doesn't exist.